README.md

```diff
@@ -1,5 +1,10 @@
 # lira
 
+[![Documentation Status](https://readthedocs.org/projects/lira/badge/?version=latest)](https://lira.python.ec/en/latest/?badge=latest)
+[![codecov.io](https://codecov.io/github/pythonecuador/lira/coverage.svg?branch=master)](https://codecov.io/github/pythonecuador/lira?branch=master)
+![tests](https://github.com/pythonecuador/lira/workflows/tests/badge.svg)
```

Comment:
This should link to https://github.com/pythonecuador/lira/actions?query=workflow%3Atests+branch%3Amaster

Reply:
Ok, update the links for test and pypi badges.